feat(changelog): block-storage-added-low-latency-offers-integration-in-c 2024-04-03 (#3009)
scaleway/docs-content: changelog/april2024/2024-04-03-block-storage-added-low-latency-offers-integration-in-c.mdx

---

> title: Low Latency offers integration in Cockpit

Reviewer: ```suggestion
title: Block Storage Low Latency is available with Cockpit
```

status: added
author:
    fullname: 'Join the #block-storage channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-04-03
category: storage
product: block-storage
---

> A new dashboard dedicated to block storage Low Latency offers is now available.

Reviewer 2: ```suggestion
A new dashboard dedicated to Block Storage Low Latency offers is now available.
```

Not sure, but either all uppercase or all lowercase I'd say

Reviewer: I've edited your comments Bene!

> Your Low Latency block storage consumption is now available for each AZ in order to improve the management of the capacity you consum.

Reviewer 2: ```suggestion
Your Block Storage Low Latency consumption is now available for each AZ in order to improve the management of the capacity you consum.
```

> Performance information are still not available, we are still working on it.

Reviewer 2: ```suggestion
We are currently in the process of working on displaying performance information, and it will be made available as soon as the feature ready.
```
